application/views/templates/admin/template.php

```diff
@@ -13,14 +13,28 @@
 		<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
 		<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
 
-    	<script src="//code.jquery.com/jquery-3.2.1.min.js"></script>
+    <script src="//code.jquery.com/jquery-3.2.1.min.js"></script>
 		<script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
 
 		<link rel="stylesheet" type="text/css" href="<?php echo base_url(); ?>assets/go/css/go_normalize.css">		
 		<link rel="stylesheet" type="text/css" href="<?php echo base_url(); ?>assets/go/css/go_default.css">	
+		
+		
+		<script type="text/javascript">
```

Comment:
This JS should be loaded in the footer, you will find some other JS in views/templates/admin/footer.php

Also, we use .on('load') instead of document ready.  See here:  https://stackoverflow.com/questions/3698200/window-onload-vs-document-ready.   Just put this inside the one that is already there.